app/assets/stylesheets/modules/spotlight_overrides.scss

```diff
@@ -8,6 +8,9 @@ body {
   .site-title {
     margin-bottom: 0.5rem;
   }
+  small.d-none {
+    display: block!important;
```

Comment:
This seems more confusing that it needs to be.  Can we remove the d-none class?

Reply:
Yes.